README.md

```diff
@@ -228,6 +228,16 @@ Most items from the INFO command are exported,
 see [Redis documentation](https://redis.io/commands/info) for details.\
 In addition, for every database there are metrics for total keys, expiring keys and the average TTL for keys in the database.\
 You can also export values of keys if they're in numeric format by using the `-check-keys` flag. The exporter will also export the size (or, depending on the data type, the length) of the key. This can be used to export the number of elements in (sorted) sets, hashes, lists, streams, etc.
+Example:
```

Comment:
Was this added on purpose? What format is that yaml in?

Reply:
This example remains from the time this request was created. After a while, I agree it confuses rather than helps.

We successfully use `redis_exporter` in a production environment but sometimes problems with the interpretation of flags surfaced (problems from the side of a non-native speaker). I believe that the documentation in the `README.md` file does not sufficiently reflect the syntax of some application flags, while they are sufficiently described in the `--help` message, for example:
```
help: 
-count-keys string
        Comma separated list of patterns to count (eg: 'db0=production_*,db3=sessions:*'), searched for with SCAN
README:
-count-keys
 Comma separated list of patterns to count, eg: `db3=sessions:*` will count all keys with prefix `sessions:` from db `3`. db defaults to `0` if omitted.


Our real production case:
Until I opened examples and tests in the repository in mid-2019, we naively believed that this line `-check-keys db0=queue1,queue2,queueN` allows us to monitor all keys in the specified database. Yes, DB 0 is used by default, but obviously, whole this string was a misconception (should be `-check-keys db0=queue1,db0=queueN`), but there was no explicit example with multiple keys at that time.

Reply:
Returning to the comment, i will remove this example from PR, its ok to me